src/Microsoft.OpenApi.OData.Reader/Edm/ODataPath.cs

```diff
@@ -146,8 +146,8 @@ public string GetPathItemName(OpenApiConvertSettings settings)
 
             // From Open API spec, parameter name is case sensitive, so don't use the IgnoreCase HashSet.
             // HashSet<string> parameters = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
-            HashSet<string> parameters = new HashSet<string>();
-            StringBuilder sb = new StringBuilder();
+            HashSet<string> parameters = new();
```

Comment:
If possible, let's keep it unchanged?

Reply:
what's the harm in using the newer syntax that saves code writing and removes squiggles when we're editing the code?

Reply:
I don't think it can save a lot of time. It's the same as:  

`var parameters = new HashSet<string>();`

Actually, it's no harm, but for readability.